### PR TITLE
fix TemplateUtils usage

### PR DESCRIPTION
**Pull request by shalk**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

- fix source.trim() if source is null
- call TenantUtil.getUserTenantForAcm() directly when  namespaceTmp is null

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java [hunk 1]
```diff
@@ -164,12 +164,7 @@ public static String parseNamespace(Properties properties) {
                         String.valueOf(Constants.DEFAULT_USE_CLOUD_NAMESPACE_PARSING)));
         
         if (Boolean.parseBoolean(isUseCloudNamespaceParsing)) {
-            namespaceTmp = TemplateUtils.stringBlankAndThenExecute(namespaceTmp, new Callable<String>() {
-                @Override
-                public String call() {
-                    return TenantUtil.getUserTenantForAcm();
-                }
-            });
+            namespaceTmp = TenantUtil.getUserTenantForAcm();
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java:167]: Why change this code?

**Review by KomachiSion — COMMENTED** [review]
I think this PR is no meaningful,

Reason is TemplateUtils.java will use isBlank and isEmpty to check source fist, if source is null, source.trim will not be called.

File: client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java (hunk 1 above)
Comment (shalk) [reply client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java:167]: line 160  String namespaceTmp = null ,  namespace is null here,  stringBlankAndThenExecute is  redundant

**Comment by shalk** [comment]
> isBlank



> I think this PR is no meaningful,
> 
> Reason is TemplateUtils.java will use isBlank and isEmpty to check source fist, if source is null, source.trim will not be called.

When callable.call  throw exception and source is null , npe will be throw.

File: client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java (hunk 1 above)
Comment (shalk) [reply client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java:167]: line 160 String namespaceTmp = null , namespace is null here, stringBlankAndThenExecute is redundant

**Comment by KomachiSion** [comment]
OK.

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**